src/profile/components/editingNotifications/EditingNotifications.module.css

```diff
@@ -0,0 +1,10 @@
+
+.wrapper {
+  padding-bottom: var(--spacing-m);
+  /*Fixes notification rendering above language selection*/
+  z-index: 1;
+  position: relative;
+}
+.wrapper:focus {
+ outline:none;
```

Comment:
Indentation and spacing seems little off. Think it would be a good idea to setup automated css lint script that would catch these. Maybe also prettier that would automatically format these correctly.

Reply:
Ah, we have prettier. My IDE shows an error with the plugin, so it was not run.

Reply:
Ran all css (old and new) through prettier. All changed!